pom.xml

```diff
@@ -100,6 +101,16 @@
                 <artifactId>r2dbc-spi</artifactId>
                 <version>${r2dbc-spi.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>cloud-spanner-r2dbc</artifactId>
+                <version>${r2dbc-cloud-spanner.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>cloud-spanner-spring-data-r2dbc</artifactId>
```

Comment:
Let's keep the Spring bits out of here. That's more something for Spring Boot itself.

Reply:
Makes sense; removed. 
Adding the dialect to [Spring Boot](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-dependencies/build.gradle) seems too much. We have Maven usage [documented](https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc/tree/master/cloud-spanner-spring-data-r2dbc) in the project with version; it should be sufficient.